spec/models/legal_aid_application_spec.rb

```diff
@@ -2327,9 +2327,10 @@
 private
 
   def uploaded_evidence_output
+    evidence_collection = laa.uploaded_evidence_collection.original_attachments.sort_by(&:original_filename)
     {
-      "benefit_evidence" => ["Fake Benefit Evidence 1", "Fake Benefit Evidence 2"],
-      "gateway_evidence" => ["Fake Gateway Evidence 1", "Fake Gateway Evidence 2"],
+      "gateway_evidence" => [evidence_collection.third, evidence_collection.fourth],
```

Comment:
first, third, etc is a bit of a code smell to me... it seems likely a race condition could change the creation sequence quite easily?

Reply:
@colinbruce Yeah, I get what you're saying. I did sort by filename on line 2230 and then took the order. Does that address your concern or do you still think i should make some changes?

Reply:
I'm happy to progress as-is, but we may end up getting some flickers on this 🤷‍♂️